dashboard/config/scripts/20-hour.script

```diff
@@ -1,3 +1,6 @@
+is_course true
+family_name '20-hour'
+version_year 'default'
```

Comment:
please make sure a version year dropdown does not appear on the script overview page for these courses

Reply:
Is 20-hour actually a CSF course or an hour of code script?

Reply:
> please make sure a version year dropdown does not appear on the script overview page for these courses

Tested, and there's no dropdown on /s/20-hour. For that matter, there's no dropdown on /s/coursea-2020 either.

> Is 20-hour actually a CSF course or an hour of code script?

No idea, I chose these scripts because they all had `curriuclum_umbrella 'CSF'`. This also seemed reasonable to me since it's called the "Accelerated Intro to CS Course". Should I not include this one?

Reply:
> Tested, and there's no dropdown on /s/20-hour. For that matter, there's no dropdown on /s/coursea-2020 either.
> 

That doesn't sound right. I see a dropdown on studio.code.org/s/coursea-2020:

![Screen Shot 2020-08-18 at 3 32 15 PM](https://user-images.githubusercontent.com/8001765/90572140-29103d80-e168-11ea-9fa0-d7732e822ab4.png)

Reply:
Sorry, I missed that you need to be signed in as a teacher. Confirmed that I see the dropdown for Course A, but not for 20 hour, locally:

![image](https://user-images.githubusercontent.com/6564873/90576048-e0f61880-e171-11ea-9ad9-ae3d4f1a9e4e.png)